### PR TITLE
Add textbox + event handler (key not bind)

### PR DESCRIPTION
**Pull request by NekoSelena**
The 'Enter' key binding is not handled, but it works with other keys. It's an issue to be fixed while handling the commands parsing (to get the user's input).

### CODE REVIEW

File: cubic-server/interface/LogsInterface.cpp [hunk 1]
```diff
@@ -6,23 +6,63 @@
...
+{
+  if (event->keyval == GDK_KEY_KP_Enter)
+  {
+    // Supposed to add the input to the logs (interface side only) when 'ENTER'
+    // is pressed. /!\ To do when handling command parsing.
+    m_label.set_text(m_label.get_text() + "\n" + m_Entry.get_text());
+    m_Entry.set_text("");
+    return true;
+  }
+  else if(event->keyval == GDK_KEY_Escape)
```
Comment (Trompettesib) [inline cubic-server/interface/LogsInterface.cpp:55]: The else here is useless I think

File: cubic-server/interface/LogsInterface.hpp [hunk 2]
```diff
@@ -9,8 +9,13 @@ class LogsInterface : public Gtk::ScrolledWindow {
     virtual ~LogsInterface();
     bool on_timeout();
 
+    private:
```
Comment (Trompettesib) [inline cubic-server/interface/LogsInterface.hpp:12]: Indentation are not respected here

File: cubic-server/interface/LogsInterface.cpp (hunk 1 above)
Comment (huntears) [reply cubic-server/interface/LogsInterface.cpp:55]: That is not a problem tbh

**Review by Trompettesib — APPROVED** [review]
LGTM

**Review by huntears — APPROVED** [review]

**Pull request merged**